src/main/java/de/baleipzig/iris/ui/view/neuralnetconfig/NeuralNetConfigView.java

```diff
@@ -63,6 +64,14 @@ public void bindTrainingsConfiguration(NeuralNetConfigViewModel viewModel) {
     }
 
     @Override
+    public void resetView() {
```

Comment:
gefällt mir nicht so gut. lieber presenter reset -> leeres viewmodel -> updateview (kommen bedienelemente dazu, müsste in dieser variante die bind und reset methode angepasst werden)